docs/templates/plugin.rst.j2

```diff
@@ -123,19 +123,24 @@ Parameters
                 </td>
                 {# default / choices #}
                 <td>
+		    {# Trun string like "yeS", "No" or "TruE", "fAlse" to boolean #}
```

Comment:
> Trun

Turn?

Reply:
Could you please move this to Python code?

Reply:
You could use this function 
https://github.com/ansible/ansible/blob/487cf0ee8d15b77fb98de38a14eddff10f3ec85b/lib/ansible/module_utils/parsing/convert_bool.py#L13
https://github.com/ansible/ansible/blob/d64e2912745d44536e4a52c55e7c5248972b8a87/test/units/module_utils/parsing/test_convert_bool.py#L35

Reply:
Try to use the boolean function you gave me as the filter, but there is a problem.
Some modules write documents like this:
```yaml
  recursive:
    description:
      - Recurse into directories.
    type: bool
    default: the value of the archive option
  links:
    description:
      - Copy symlinks as symlinks.
    type: bool
    default: the value of the archive option
```
So they define a bool arg but give a no bool string as the default.